src/main/sensors/battery.c

```diff
@@ -221,41 +220,42 @@ static void updateCurrentDrawn(int32_t lastUpdateAt)
 
 void updateCurrentMeter(int32_t lastUpdateAt, rxConfig_t *rxConfig, uint16_t deadband3d_throttle)
 {
-    switch(batteryConfig->currentMeterType) {
-        case CURRENT_SENSOR_ADC:
-            updateBatteryCurrent();
-
-            updateCurrentDrawn(lastUpdateAt);
-
-            break;
-        case CURRENT_SENSOR_VIRTUAL:
-            amperage = (int32_t)batteryConfig->currentMeterOffset;
-            if (ARMING_FLAG(ARMED)) {
-                throttleStatus_e throttleStatus = calculateThrottleStatus(rxConfig, deadband3d_throttle);
-                int throttleOffset = (int32_t)rcCommand[THROTTLE] - 1000;
-                if (throttleStatus == THROTTLE_LOW && feature(FEATURE_MOTOR_STOP)) {
-                    throttleOffset = 0;
+    #ifdef USE_ESC_SENSOR
+    if (feature(FEATURE_ESC_SENSOR))
```

Comment:
That is kind of a chicken and egg argument. If we remove these options from the firmware now, then it is obvious that the configurator will not any more be able to support them. If we leave them in, it will take very little in `serial_cli.c` to make them configurable from the configurator.

I suspect if we don't add the options, this will just result in nobody using ESC feedback, since the sensors on the ESCs will not be more accurate than the ones on the FC, and the ones on the FC can be calibrated.